sql/src/test/java/org/apache/druid/quidem/DruidQuidemCommandHandler.java

```diff
@@ -245,6 +269,159 @@ static class DruidPlanCommand extends AbstractRelPlanCommand
     }
   }
 
+  static class HintPlanCommand extends AbstractRelPlanCommand
+  {
+    HintPlanCommand(List<String> lines, List<String> content)
+    {
+      super(lines, content, DruidHook.DRUID_PLAN);
+    }
+
+    @Override
+    protected String getString(RelNode node)
+    {
+      final List<String> hintsCollect = new ArrayList<>();
```

Comment:
note: this could be created as a field in the `HintCollector`

Reply:
Created a method inside HintCollecter to get the hints converted as String